packages/core/next.config.js

```diff
@@ -8,6 +8,8 @@ const nextConfig = {
   /* config options here */
   images: {
     domains: [`${storeConfig.api.storeId}.vtexassets.com`],
+    deviceSizes: [320, 768, 1280, 1440],
+    imageSizes: [34, 68, 154],
```

Comment:
https://nextjs.org/docs/api-reference/next/image#device-sizes
Aims to provides `srcSet` for Next images.